src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -175,6 +175,13 @@ public function denormalize($data, $class, $format = null, array $context = arra
         }
 
         $allowedAttributes = $this->getAllowedAttributes($class, $context, true);
+
+        if (false === $allowedAttributes && $this->classMetadataFactory) {
```

Comment:
It's probably useless to do this if `$context[self::ALLOW_EXTRA_ATTRIBUTES]` is not set.

Reply:
Right. Thanks :)

Reply:
More presicesly if it's not set to false. True is default. :-)